docs/src/implemented-proposals/rent.md

```diff
@@ -18,13 +24,19 @@ Accounts whose balance is insufficient to satisfy the rent that would be due sim
 
 A percentage of the rent collected is destroyed. The rest is distributed to validator accounts by stake weight, a la transaction fees, at the end of every slot.
 
-## Read-only accounts
+Finally, rent collection happens according to the protocol-level account updates like the rent distribution to validators, meaning there is no corresponding transaction for rent deductions. So, rent collection is rather invisible, only implicitly observable by a recent transaction or predetermined timing given its account address prefix.
```

Comment:
I hope this line "rather invisible" doesn't last too long. I think https://github.com/solana-labs/solana/issues/10088 could help!

Reply:
Yeah, I think so too. :)